src/Entity/ContentMagicTrait.php

```diff
@@ -164,13 +189,7 @@ public function magicTitle(): string
             $titleParts[] = $this->get($field);
         }
 
-        $title = trim(implode(' ', $titleParts));
-
-        if (empty($title)) {
-            return '(untitled)';
-        }
-
-        return $title;
+        return trim(implode(' ', $titleParts));
```

Comment:
I see why you did this, but this means i'll have to fix it in the Backend UI. 

It's better though, so i'll put it on my list. :-)